### ISSUE TITLE
CVR Inquiry for Provided Question

### ISSUE THREAD

**Issue by FurlanNick** [gryphonshafer/Quizzing-Rule-Book#201]
Hello, 

I just had a question in regards to a CVR issue we had with our District quiz meet. 

A quizmaster stated "According to Ephesians 1 Verse 6: His glor"

The quizzers than answered the question. The quizmaster asked for their question. The quizzers responded with "Glorious what?"

Would this be a provided correctly? I recall there was a statement that you need to provide what was stated previously by the quiz master. Just curious on the understanding of this situation.

Thanks,

Nick

**Comment by JoshJetto** [comment]
Hey Nick, there is nothing in the rulebook that states the quizzer must repeat what the quizmaster said in giving their question, only that the question the quizzer gives be of the exact same question type and similar to the question the quizmaster was asking, so "Glorious what?" would still be a CVR and is similar to "His glorious what?" so the quizzer should have been called correct...In revisiting the rulebook, the rulebook does not actually explicitly state that a quizzer must give the question if they prejump a CR or CVR; it only says "If they must give the question..." in the Reference section without ever specifying the scenario in which they "must" give the question. And it is only incidentally referred to in the Quizmaster Prompts section, where it says what a quizmaster must do without specifying what a quizzer must do in order to be called correct.

**Comment by jttower** [comment]
I agree with Josh that "Glorious what" would be acceptable. The assumption would be that the word "glorious" is the pertinent word here in the question and directly precedes the answer, which is what causes it to be a REF question.  

However I think the rulebook does state that a quizzer must give the question.  I believe this would fall under the more broad rule under 5.2.3 Correct that "If the quizmaster did not complete the reading of the question, the quizzer's answer must include the information in the remainder of the question as well as the answer." It is because of this rule that I think it is referenced in the REF (2.2.2.3) and QM Prompts (3.2.1.1) sections.  I agree that it could be confusing, however, that these sections talk about a rule that isn't stated until later in the rulebook (5.2.3 Correct).  Perhaps changing the order of the sections should be considered?

**Comment by levikoral** [comment]
In my understanding of what the rule book says about this question type, if a quizzer jumps before the CVR is finished, they must finish the question after giving the answer (or the individual provides the question before the answer, which the rulebook does not state is wrong). The only instance where there is a proper order of answering the question is when the whole reference is not given by the quizmaster. If that occurs, the quizzer must provide the answer, then finish the reference, and finally give the question.

**Comment by levikoral** [comment]
However, the first instance is being talked about, and it is required that the quizzer give the necessary "information in both the question and the answer." It is then that the quizmaster asks for the question if "the complete question was not read" (excerpts taken from section 3.2.1.1.2 Reference Questions). Although it does not explicitly say whether the person must give the exact question that the quizmaster would have given, I think it is safe to assume that with the given information the quizzer should be able to conclude what the question would be. The reason I say this is because the rulebook does not state that question _can_ deviate from what was written on the quizmaster's question sheet. However, if this is being argued about, I would advise that the rulebook be altered to avoid further clarification- especially if a district is newer, and if a question type is difficult to understand, then a certain amount of confusion may arise.

**Comment by scrobiculatus** [comment]
The question "Glorious what?" doesn't really include all the information in "His glorious what?" I'm pretty sure the rule for alternate questions is that they still have to contain all the information in the written question and answer, but "Glorious what?" doesn't suggest that the thing that is glorious is his.
